posix-mapper/src/main/java/org/opencadc/posix/mapper/db/InitializeMappingDatabase.java

```diff
@@ -0,0 +1,119 @@
+ *  License as published by the          License” telle que publiée
+ *  Free Software Foundation,            par la Free Software Foundation
+ *  either version 3 of the              : soit la version 3 de cette
+ *  License, or (at your option)         licence, soit (à votre gré)
+ *  any later version.                   toute version ultérieure.
+ *
+ *  OpenCADC is distributed in the       OpenCADC est distribué
+ *  hope that it will be useful,         dans l’espoir qu’il vous
+ *  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+ *  without even the implied             GARANTIE : sans même la garantie
+ *  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+ *  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+ *  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+ *  General Public License for           Générale Publique GNU Affero
+ *  more details.                        pour plus de détails.
+ *
+ *  You should have received             Vous devriez avoir reçu une
+ *  a copy of the GNU Affero             copie de la Licence Générale
+ *  General Public License along         Publique GNU Affero avec
+ *  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+ *  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+ *                                       <http://www.gnu.org/licenses/>.
+ *
+ *
+ ************************************************************************
+ */
+
+package org.opencadc.posix.mapper.db;
+
+import ca.nrc.cadc.db.version.InitDatabase;
+import ca.nrc.cadc.util.MultiValuedProperties;
+import org.opencadc.posix.mapper.web.PosixInitAction;
+
+import javax.sql.DataSource;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ * Initialize the underlying database with necessary tables.
+ */
+public class InitializeMappingDatabase extends InitDatabase {
+
+    public static final String MODEL_NAME = "posix-mapping";
+    public static final String MODEL_VERSION = "0.1";
+    public static final String PREV_MODEL_VERSION = "0.0";
+
+    static String[] CREATE_SQL = new String[] {
+            "mapping.modelversion.sql",
+            "mapping.model.sql"
+    };
+
+    public InitializeMappingDatabase(DataSource dataSource, String database, String schema) {
+        super(dataSource, database, schema, InitializeMappingDatabase.MODEL_NAME,
+              InitializeMappingDatabase.MODEL_VERSION, InitializeMappingDatabase.PREV_MODEL_VERSION);
+
+        this.createSQL.addAll(Arrays.stream(InitializeMappingDatabase.CREATE_SQL).collect(Collectors.toList()));
+    }
+
+    @Override
+    public List<String> parseDDL(String fileName, String schema, String tag) throws IOException {
+        final MultiValuedProperties configuration = PosixInitAction.getConfig();
+        return super.parseDDL(fileName, schema, tag).stream()
```

Comment:
ah, clever. This will of course mean that one can only set the min values once rather than change them later with `alter sequence`... but that's probably fine for now.